components/Careers.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { FunctionComponent } from "react";
+import ICareersProps from "types/CareersProps";
+import JobOpportunityCard from "./JobOpportunityCard";
+import SectionTitle from "./SectionTitle";
+import Button from "./Button";
+import MockJobOpportunities from "../mocks/JobOpportunities";
+
+/**
+ * Button component for user interaction
+ */
+const Careers: FunctionComponent<ICareersProps> = ({ ...props }) => {
+  return (
+    <div
+      className="relative max-w-6xl p-4 m-auto md:max-xl:w-9/12 md:max-lg:w-11/12"
+      {...props}
+      data-cy="careers"
+    >
+      <SectionTitle
+        sectionTitle={props.sectionTitle}
+        description={props.description}
+        mode={props.mode}
+      />
+
+      <div className="flex flex-wrap w-full mt-8 mb-10">
+        {MockJobOpportunities.slice(0, 3).map((vacancy) => (

```

Comment:
These shouldn't be here, please place the mock array on the storybook file, not in the component.

Reply:
Done. It was necessary to change the MoreOpportunities component too, it also used this Mock created and as the Mock was deleted the component broke.